pyta_server/routes.py

```diff
@@ -0,0 +1,30 @@
+import os
+import secrets
+import psycopg2
+from flask import request
+from pyta_server import app, db
+from pyta_server.models import Uploads
+
+@app.route('/', methods=['POST'])
+def receive():
+    rand_hex = secrets.token_hex(8)
+    src_f = request.files['source']
```

Comment:
This part is definitely looking better, except don't assume that these files will always be in the request. (You might, after all, receive a bad request.) You can use `dict.get` to retrieve a value from a dictionary, but have it return `None` when the key doesn't appear. Then check for either of these values being `None` and you can return a "bad request" message if this occurs.